coalib/results/AbsolutePosition.py

```diff
@@ -27,6 +27,90 @@ def __init__(self,
     def position(self):
         return self._position
 
+    def __le__(self, other):
+        """
+        Test whether ``self`` is behind or equals the other
+        ``AbsolutePosition``.
+
+        :param other: ``AbsolutePosition`` to compare with.
+        :return:      Whether this ``AbsolutePosition`` is behind the other
+                      one or the same.
+        """
+        if not isinstance(other, self.__class__):
+            return False
+
+        if self._text != other._text:
+            return False
+
+        return self.position <= other.position
+
+    def __ge__(self, other):
+        """
+        Test whether ``self`` is ahead of or equals the
+        other ``AbsolutePosition``.
+
+        :param other: ``AbsolutePosition`` to compare with.
+        :return:      Whether this ``AbsolutePosition`` is ahead of the other
```

Comment:
how about  something of the form: 
``True`` when .... ``False`` when or raise a ``TypeError``